### PR TITLE
No longer setting CurrentUnits = PreferredUnits (issue 95)

### LINKED ISSUE #95: Fix missing `%` unit for SUTRA pump efficiency
See https://github.com/softwareengineerprogrammer/GEOPHIRES-X/pull/3/files#r1463954946

### PR DESCRIPTION
**Pull request by jeffbourdier**
This should fix #95.  Note that examples with input units specified as "feet" were changed to "ft" for compatibility with [Units.py](https://github.com/NREL/GEOPHIRES-X/blob/main/src/geophires_x/Units.py#L113).

### CODE REVIEW

File: tests/examples/Fervo_Project_Cape.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@
 Reservoir Model, 1
 Reservoir Volume Option, 1
 Reservoir Density, 2800
-Reservoir Depth, 8500 feet, -- https://pangea.stanford.edu/ERE/db/GeoConf/papers/SGW/2024/Fercho.pdf
```
Comment (softwareengineerprogrammer) [inline tests/examples/Fervo_Project_Cape.txt:7]: This would be a backwards-incompatible change - can we preserve `feet` as a valid unit?

**Review by softwareengineerprogrammer — CHANGES_REQUESTED** [review]
Actions are failing - looks like example output(s) need to be regenerated to include percent unit (which is a good thing) https://github.com/NREL/GEOPHIRES-X/actions/runs/11769618204/job/32780887255?pr=307#step:5:314

File: tests/examples/Fervo_Project_Cape.txt (hunk 1 above)
Comment (jeffbourdier) [reply tests/examples/Fervo_Project_Cape.txt:7]: If I understand [the Units classes](https://github.com/NREL/GEOPHIRES-X/blob/main/src/geophires_x/Units.py) correctly, a given unit can have only one value (e.g., in `LengthUnit`, the value for `FEET` is "ft").  The only way I see to allow multiple values for a given unit would be to add a special case into [LookupUnits](https://github.com/NREL/GEOPHIRES-X/blob/main/src/geophires_x/Parameter.py#L747), but this strikes me as a hack.  Is there another (or a better) way?

Comment (softwareengineerprogrammer) [reply tests/examples/Fervo_Project_Cape.txt:7]: We might be able to add an entry in https://github.com/NREL/GEOPHIRES-X/blob/main/src/geophires_x/GEOPHIRES3_newunits.txt

**Comment by softwareengineerprogrammer** [comment]
Note: this was closed in favor of https://github.com/NREL/GEOPHIRES-X/pull/310

**Pull request closed without merging**